collect_app/src/main/res/layout/storage_migration_banner.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/storageMigrationBanner"
+    android:orientation="vertical"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="?colorSurface"
+    android:visibility="gone">
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        android:paddingStart="@dimen/margin_standard"
+        android:paddingEnd="@dimen/margin_standard"
+        android:paddingTop="@dimen/margin_standard">
+
+        <TextView
+            android:id="@+id/storageMigrationBannerText"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:textColor="?colorOnSurface">
+        </TextView>
+
+        <Button
+            android:id="@+id/storageMigrationBannerDismissButton"
+            style="@style/Widget.MaterialComponents.Button.TextButton"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="end"
+            android:text="@string/scoped_storage_dismiss"
+            android:textColor="?colorAccent"
+            android:paddingTop="12dp"
+            android:paddingBottom="@dimen/margin_small"
+            android:textStyle="bold"
```

Comment:
I prefer having it bold otherwise it's like:
![Screenshot_2020-02-21-09-53-41](https://user-images.githubusercontent.com/3276264/75014940-c3c4e380-5487-11ea-88d4-54d4c23bc6d4.png)

the sample from material design is with bold:
![image](https://user-images.githubusercontent.com/3276264/75014973-d4755980-5487-11ea-91cf-1515bd0518bd.png)

Reply:
Hmmm yeah it feels to me like the button examples they have are using a heavier font than I can seem to get access to using the Material Component styles. We're looking at changing the Main Menu buttons to be Material Buttons which means us thinking about these styles so I'm happy to just go with bold on this for now if you prefer it and we can play with different options then.

So this is easier for us to play with globally could you make a new style in `typography.xml` for the buttons (`TextAppearance.Collect.Button`) that inherits from `TextAppearance.MaterialComponents.Button` with the bold `textStyle`. If we set it as the `textAppearanceButton` in our themes that will make the bold style apply to the `MaterialButton` objects (with the `TextButton` style here of course).

Reply:
You mean like https://github.com/opendatakit/collect/pull/3623/commits/713f6865a307681891c93ef04c816a260ecb51ba?
Is this ok?

Reply:
Not exactly. On the button itself (the `MaterialButton`) you still have:

```xml
style="@style/Widget.MaterialComponents.Button.TextButton"
```

Then in `typography.xml` we add:

```xml
<style name="TextAppearance.Collect.Button" parent="TextAppearance.MaterialComponents.Button">
        <item name="android:textStyle">bold</item>
</style>
```

Then in `light_theme.xml` and `dark_theme.xml` we change the `textAppearanceButton` attribute so that we have:

```xml
<item name="textAppearanceButton">@style/TextAppearance.Collect.Button</item>
```

The Material Component styles use the `textAppearanceButton` theme attribute automatically.

Reply:
Ok done.